<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="dell" ignored="3" name="Default test" tests="1" failures="1" timestamp="31 Aug 2018 02:01:10 GMT" time="74.652" errors="0">
  <testcase name="profile" time="40.075" classname="testPackage.TestClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at javax.imageio.ImageIO.write(Unknown Source)
at utilityPackage.ScreenShot.captureScreenshot(ScreenShot.java:24)
at pagesPackage.MyProfile.Validateprofile(MyProfile.java:89)
at pagesPackage.MyProfile.Verifyprofile(MyProfile.java:60)
at testPackage.TestClass.profile(TestClass.java:73)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- profile -->
  <testcase name="Account" time="0.0" classname="testPackage.TestClass">
    <ignored/>
  </testcase> <!-- Account -->
  <testcase name="subscription" time="0.0" classname="testPackage.TestClass">
    <ignored/>
  </testcase> <!-- subscription -->
  <testcase name="Login" time="0.0" classname="testPackage.TestClass">
    <ignored/>
  </testcase> <!-- Login -->
</testsuite> <!-- Default test -->
